pkg/trace/agent/normalizer_test.go

```diff
@@ -188,6 +189,28 @@ func TestNormalizeNoTraceID(t *testing.T) {
 	assert.Equal(t, tsDropped(&info.TracesDropped{TraceIDZero: 1}), ts)
 }
 
+func TestNormalizeComponent2Name(t *testing.T) {
+	ts := newTagStats()
+	assert := assert.New(t)
+
+	defer testutil.WithFeatures("component2name")()
+
+	t.Run("with", func(t *testing.T) {
```

Comment:
nit: it would be nice to add a test case when the feature is turned off.

Reply:
Good point. Added.